src/journal-remote/microhttpd-util.h

```diff
@@ -65,15 +65,23 @@ void microhttpd_logger(void *arg, const char *fmt, va_list ap) _printf_(2, 0);
 int mhd_respond_internal(
                 struct MHD_Connection *connection,
                 enum MHD_RequestTerminationCode code,
+                char *encoding,
                 const char *buffer,
                 size_t size,
                 enum MHD_ResponseMemoryMode mode);
 
-#define mhd_respond(connection, code, message)                  \
-        mhd_respond_internal(                                   \
-             connection, code,                                  \
-             message "\n",                                      \
-             strlen(message) + 1,                               \
+#define mhd_respond(connection, code, message)         \
+        mhd_respond_internal(                          \
+             connection, code, NULL,                   \
+             message "\n",                             \
+             strlen(message) + 1,                      \
+             MHD_RESPMEM_PERSISTENT)
```

Comment:
i understand this is a pe-existing problem, but `connect` and `code` should be enclosed in `()`. if you touch this, plesae fix.